wrappers/python/simtk/unit/unit_definitions.py

```diff
@@ -157,7 +157,7 @@
 moles = mole = Unit({mole_base_unit: 1.0})
 
 single_item_amount_base_unit = BaseUnit(amount_dimension, "item", "")
-mole_base_unit.define_conversion_factor_to(single_item_amount_base_unit, 6.0221417930e23)
+mole_base_unit.define_conversion_factor_to(single_item_amount_base_unit, 6.02214076e23)
```

Comment:
Can we add a note in this file that we updated this to CODATA 2018, in case someone ever copies this file?

Also, a number of the other conversion factors in this file have not been updated to CODATA 2018.
* [Planck time](https://physics.nist.gov/cgi-bin/cuu/Value?plkt)
* [Planck length](https://physics.nist.gov/cgi-bin/cuu/Value?plkl|search_for=planck)
* [Bohr radius](https://physics.nist.gov/cgi-bin/cuu/Value?bohrrada0|search_for=bohr)

Hm, it actually looks like _all_ of these conversion factors need updating!